pkg/controller/volume/persistentvolume/metrics/metrics.go

```diff
@@ -110,6 +118,23 @@ var (
 		[]string{"plugin_name", "operation_name"})
 )
 
+// volumeCount counts by PluginName, StorageClass and VolumeMode.
+type volumeCount map[string]map[string]map[string]int
```

Comment:
Can we improve readability of this structs, we can do perhaps define named type (like type pluginName string) and then use it in aggregate type.

Reply:
It's simpler now and looks completely like the metrics in the volume manager. So I hope it's enough.